src/main/java/edu/kis/powp/jobs2d/command/visitor/IElement.java

```diff
@@ -0,0 +1,5 @@
+package edu.kis.powp.jobs2d.command.visitor;
+
+public interface IElement {
```

Comment:
I would resign from this interface (add accept directly to DriverCommand). The name and contents is not useful and self-explained.